shared/middleware/net_sim_api.rb

```diff
@@ -178,32 +178,75 @@ def owns_shard?(shard_id)
   #
   # POST /v3/netsim/<shard-id>/<table-name>
   #
-  # Insert a new row.
+  # Insert a new row or rows.
```

Comment:
Could you document the allowed POST body formats, now that this is getting more complicated.